src/TortoiseProc/AppUtils.cpp

```diff
@@ -3268,7 +3268,44 @@ BOOL CAppUtils::MergeAbort()
 {
 	CMergeAbortDlg dlg;
 	if (dlg.DoModal() == IDOK)
-		return Reset(_T("HEAD"), dlg.m_ResetType + 1);
+	{
+		if (dlg.m_ResetType == 0)
+		{
+			CString cmd;
+			if (GetMsysgitVersion() < 0x01070600)
+				cmd = _T("git.exe reset --merge");
+			else
+				cmd = _T("git.exe merge --abort");
+
+			CProgressDlg progress;
+			progress.m_GitCmd = cmd;
+
+			progress.m_PostCmdCallback = [&](DWORD status, PostCmdList& postCmdList)

```

Comment:
I think so, because of the [issue 1608](https://code.google.com/p/tortoisegit/issues/detail?id=1608).